pypesto/store/save_to_hdf5.py

```diff
@@ -315,3 +320,110 @@ def write_result(result: Result,
     if sample:
         pypesto_sample_writer = SamplingResultHDF5Writer(filename)
         pypesto_sample_writer.write(result, overwrite=overwrite)
+
+
+def save_objective_infos(
+        objective: Union[AmiciObjective,
+                         AggregatedObjective,
+                         Objective],
+        f: h5py.File
+):
+    """
+    Saves the Objective Infos in a Dictonary.
```

Comment:
```suggestion
    """Save the Objective Infos in a Dictonary.
```
infinitive/imperative